### PR TITLE
Add Eclipse Ditto Helm Chart

### PR DESCRIPTION
**Pull request by axdotl**
@rpahli @monotek @fabian-schlegel This is still WIP, but I'd like to get your feedback for the current state.

### CODE REVIEW

File: charts/ditto-digital-twins/templates/connectivity-networkpolicy.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  ingress:
+  # Allow access from concierge
+  - from:
+    - podSelector:
+        matchLabels:
+          app.kubernetes.io/name: {{ include "ditto-digital-twins.name" . }}-concierge
+          app.kubernetes.io/instance: {{ .Release.Name }}
+    ports:
+    - protocol: TCP
+      port: 8080
```
Comment (rpahli) [inline charts/ditto-digital-twins/templates/connectivity-networkpolicy.yaml]: is it always 8080?

File: charts/ditto-digital-twins/templates/gateway-deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    {{- end }}
+      containers:
+        - name: {{ .Chart.Name }}-gateway
+          image: "{{ .Values.gateway.image.repository }}:{{ .Values.gateway.image.tag }}"
+          imagePullPolicy: {{ .Values.gateway.image.imagePullPolicy }}
+          env:
+            - name: K8S_NAMESPACE
+              value: {{ .Release.Namespace }}
+            - name: ENABLE_DUMMY_AUTH
+              value: "true"
```
Comment (rpahli) [inline charts/ditto-digital-twins/templates/gateway-deployment.yaml]: values file?

File: charts/ditto-digital-twins/templates/role.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+kind: Role
```
Comment (rpahli) [inline charts/ditto-digital-twins/templates/role.yaml]: can we add a rbac.enabled property?

File: charts/ditto-digital-twins/templates/swaggerui-deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "ditto-digital-twins.name" . }}-swaggerui
+      app.kubernetes.io/instance: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "ditto-digital-twins.name" . }}-swaggerui
+        app.kubernetes.io/instance: {{ .Release.Name }}
+        actorSystemName: {{ .Values.akka.actorSystemName }}
```
Comment (rpahli) [inline charts/ditto-digital-twins/templates/swaggerui-deployment.yaml]: pls remove akka label ... not needed here

**Comment by axdotl** [comment]
I'm almost done.
From my point of view there are some open points (mentioned in README) which I'd like to cover in a later PR.

**Comment by monotek** [comment]
@axdotl ci seems to be fine now. Have you changed anything beside timeout?

**Comment by axdotl** [comment]
No, only changed timeouts. We'll see whether next builds succeeds :)

**Pull request merged**